internal/server/dump.go

```diff
@@ -71,6 +76,64 @@ func (s *Server) Dump(ctx context.Context, req *daemon.DumpReq) (*daemon.DumpRes
 	if utils.PathExists(resp.Path) {
 		log.Info().Str("path", resp.Path).Str("type", req.Type).Msg("dump successful")
 		resp.Messages = append(resp.Messages, "Dumped to "+resp.Path)
+
+		cedanaUrl := config.Global.Connection.URL
+		cedanaAuthToken := config.Global.Connection.AuthToken
```

Comment:
Checkpoint upload should be part of the existing filesystem/streamer middleware, or be a middleware of its own. Also, should we create a client each time there's a dump request? Can this not be part of server/cedanaRoot initialization?